SWLOR.Game.Server/Data/Migrations/2019-08-18.1.sql

```diff
@@ -0,0 +1,25 @@
+﻿-- Throwing Axes
+INSERT INTO dbo.CraftBlueprint ( ID ,CraftCategoryID ,BaseLevel ,ItemName ,ItemResref ,Quantity ,SkillID ,CraftDeviceID ,PerkID ,RequiredPerkLevel ,IsActive ,
+MainComponentTypeID ,MainMinimum ,SecondaryComponentTypeID ,SecondaryMinimum ,TertiaryComponentTypeID ,TertiaryMinimum ,EnhancementSlots ,MainMaximum ,
+SecondaryMaximum ,TertiaryMaximum ,BaseStructureID )
+VALUES ( 683 ,12 ,2 , N'Basic Axe' , N'axe_b' ,  1 , 12 , 2 , NULL , 0 ,  1 , 4 ,  1 ,  8 ,  1 ,  0 ,  0 ,  1 ,  2 ,  2 ,  0 ,  NULL)
```

Comment:
Can we call these Throwing Axe instead of just Axe? I think it's more clear to NWN players.